pyproject.toml

```diff
@@ -143,3 +143,36 @@ markers = [
     "needs_solver"
 ]
 testpaths = "idaes_examples"
+
+[tool.typos.files]
+extend-exclude = [
+    "*.svg",
+    "*.json",
+    "*.css",
+    "*.yml",
+]
+
+[tool.typos.default.extend-words]
+# Ignore IDAES
+IDAES = "IDAES"
+# Ignore HDA, assumes it's on purpose and not a typo of "had"
+HDA = "HDA" 
+# Ignore Attemp - assume it is abbreviating attemperator
+Attemp = "Attemp"
+# Ignore ba in hexadecimal values, or block names
+ba = "ba"
```

Comment:
Is this ignore still required now that we have the regex later to exclude the hex values in the id field? If not, we should remove it.

Reply:
Tested locally, it catches block names e.g. `ba23` and also atomic element usage of `Ba` for barium. Those are flagged as typos if this ignore line is taken out.

Reply:
First, where is `ba23` used? That does not look like a very good name for a block.
Catching `Ba` for barium is important, but we should update the comment to indicate this.

Reply:
OK - we should update the comment however to indicate that it is there to catch these (as otherwise someone else might as k the same question I did).